resources/views/mail/markdown/checkout-accessory.blade.php

```diff
@@ -1,5 +1,5 @@
 @component('mail::message')
-# {{ trans('mail.hello') }} {{ $target->present()->fullName() }},
+# {{ trans('mail.hello') }} {{ $target->assignedto->present()->fullName() }},
```

Comment:
We'll need to check for nulls here - tho I'm not sure why it would be null, except for possibly different methods?